src/panels/config/automation/trigger/types/ha-automation-trigger-state.ts

```diff
@@ -94,6 +95,14 @@ export class HaStateTrigger extends LitElement implements TriggerElement {
           name: "from",
           selector: {
             state: {
+              extra_options: (attribute
+                ? []
+                : [
+                    {
+                      label: "Any state (ignoring attribute changes)",
```

Comment:
How should this be localized? Do I pass hass object into the memoize function? That doesn't seem optimal. Or should it be translated elsewhere?